src/styles/blocks/catalog.scss

```diff
@@ -5,6 +5,7 @@
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
+  perspective: 400px;
```

Comment:
The 'perspective' property seems unnecessary. Please remove it if it's not required for the task.